README.md

```diff
@@ -95,6 +96,52 @@ const server = new PolarisServer({
 });
 ```
 
+### Warnings
+
+In order to have the ability of warnings, which returned in the extensions of the response, you will need to supply them to
+polaris. you can supply the warnings through the `customContext` - when you create the `customContext`(of type `PolarisGraphQLContext`),
+set the `returnedExtensions` with the relevant warnings. let's see an example:
+
+```typescript
+const warnings: PolarisWarning[] = ['warning 1', 'warning 2'];
+const extensions: PolarisExtensions = { warnings: warnings };
```

Comment:
you should add set your warnings like this:
`customContext.returnedExtensions.warnings.push('warning 1')`

Reply:
Show example of a resolver that adds a warning, which is not using custom context.
Use normal context, to make it less complicated.